### DISCUSSION TITLE
Styling of ErrorMessage component using styled-components

### DISCUSSION THREAD

**Discussion by qasimalyas** [react-hook-form/react-hook-form#1681] [General] [answered]
I'm having an issue when using the `ErrorMessage` component and customising it with styled-component.

Here's a demo. https://codesandbox.io/s/patient-sky-5jekz?file=/src/App.js

You can see that I created a styled component `StyledError` that I should be able to use instead with the attached styles. Everything I've read suggests that this should work.

**Comment by bluebill1049** [comment, marked as answer]
I suspect this is due to `as` props, I would recommend to do things the other way around. make a style component and pass it to `as`.
